### ISSUE TITLE
where is eco and should we add these codes for FYPO use

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#1035]
I can't find where I can search ECO?

Should we add

recombination assay
and 
random spore analysis
as options for use with FYPO?

https://curation.pombase.org/pombe/curs/903cb37e16a1d237/ro/

**Comment by mah11** [comment]
> I can't find where I can search ECO?

http://evidenceontology.org/ - "Visualize ECO" in menu

> Should we add
> 
> recombination assay
> and
> random spore analysis
> as options for use with FYPO?

Short answer: I vote no.

Explanation: In random spore analysis, spores are picked at random (as opposed to tetrad dissection) and put on plates to see if they grow. Usually any colonies that do grow are also transferred to various selective media to see which markers they do and don't have, which allows you to infer their genotype, and see if any genotypes never appear among the colony-forming spores. I don't see any reason to treat it any differently than any of the several other methods that we've lumped into "cell growth assay".

For recombination assays we've used evidence that describes how the recombination was detected -usually reporter gene or gel electrophoresis, depending on the experiment (I'm pretty sure Antonia has done a lot more such papers than I have). I think doing it this way is actually more informative than just saying "recombination assay", because the "recombination" FYPO term should provide a clue that recombination was assayed.

Unless we're going to add "tetrad analysis", "growth curve", "spot assay", and probably some I'm not thinking of under cell growth, there's no basis for adding one type. Similarly, in theory we could "reporter gene assay for recombination", "gel electrophoresis assay for recombination" and I don't know what else, but we've never been so inclined thus far.

To sum up, I don't think it makes sense to add more specificity to the FYPO evidence selection piecemeal. We should either keep the options we've got, and live with the loss of specificity, or go straight to deploying ECO (and live with the fact that 2700 papers' worth of retrofitting won't get done for ages).

**Comment by ValWood** [comment]
OK, makes sense. I think this once came up in a curation workshop. Might be worth shuttling the method away into the comment...
